duo-client/src/main/java/com/duosecurity/client/Util.java

```diff
@@ -86,4 +88,23 @@ public static CertificatePinner createPinner(String apiHost, String[] caCerts) {
 
     return pinner;
   }
+
+  /**
+   * Create hash byte array of message
+   *
+   * @param algorithm   The algorithm used to create the hash
+   * @param message     The text to create the hash
+   * @return a byte array
+   */
+  public static byte[] hash(String algorithm, String message) {
+      MessageDigest digest;
+      try {
+        digest = MessageDigest.getInstance(algorithm);
+        byte[] encodedhash = digest.digest(message.getBytes(StandardCharsets.UTF_8));
+        return encodedhash;
+      } catch (NoSuchAlgorithmException e) {
+        e.printStackTrace();
+        return new byte[0];
```

Comment:
@AaronAtDuo 
To follow previous convo about exception, there is another place similar exception need to be catch is `signHMAC`, the other method just return `""` , so maybe it is okay to return empty byte[] here too? 
I also added `e.printStackTrace()` not sure it that's the right thing to do.

Reply:
> I also added e.printStackTrace() not sure it that's the right thing to do.
Nope :)
nobody will see it

Reply:
I don't love the plan of returning 'nothing' but let's stay consistent for now and revisit if we need to.